README.md

```diff
@@ -56,3 +56,67 @@ hls_model.build()
 #Print out the report if you want
 hls4ml.report.read_vivado_report('my-hls-test')
 ```
+
+# Citation
+If you use this software in a publication, please cite the software
+```bibtex
+@software{vloncar_2021_5680908,
+  author       = {Loncar, Vladimir and others},
+  title        = {fastmachinelearning/hls4ml: coris},
+  year         = 2021,
+  publisher    = {Zenodo},
+  version      = {v0.6.0},
+  doi          = {10.5281/zenodo.5680908},
+  url          = {https://github.com/fastmachinelearning/hls4ml}
+}
+```
+and first publication:
+```bibtex
+@article{Duarte:2018ite,
+    author = "Duarte, Javier and others",
+    title = "{Fast inference of deep neural networks in FPGAs for particle physics}",
+    eprint = "1804.06913",
+    archivePrefix = "arXiv",
+    primaryClass = "physics.ins-det",
+    reportNumber = "FERMILAB-PUB-18-089-E",
+    doi = "10.1088/1748-0221/13/07/P07027",
+    journal = "JINST",
+    volume = "13",
+    number = "07",
+    pages = "P07027",
+    year = "2018"
+}
+```
+Additionally, if you use specific features developed in later papers, please cite those as well. For example, CNNs:
+```bibtex
+@article{Aarrestad:2021zos,
+    author = "Aarrestad, Thea and others",
+    title = "{Fast convolutional neural networks on FPGAs with hls4ml}",
```

Comment:
Hmm, while this paper has CNN in it's title, it describes a deprecated implementation that no one uses. The one people use is either [this](https://iopscience.iop.org/article/10.1088/2632-2153/ac9cb5) one or the new io_parallel one that is not published anywhere

Reply:
It's a valid point. I added the other CNN paper. At the end of the day, it's up to the user to cite whatever they want, and maybe once the new implementation is published somewhere we can point to that one instead.